include/copentime/util.h

```diff
@@ -39,3 +41,12 @@ inline opentime::TimeTransform CTimeTransform_to_CppTimeTransform(TimeTransform
     opentime::RationalTime offset = CRationalTime_to_CppRationalTime(timeTransform.offset);
     return opentime::TimeTransform(offset, timeTransform.scale, timeTransform.rate);
 }
+
+inline char* CppString_to_CString(std::string const& s)
+{
+    const size_t size = s.size();
```

Comment:
could just use strdup here... see also _CRT_SECURE_NO_WARNINGS
also no need to inline, the cost is in the heap, not the function calls